libs/IO/ImageTIFF.cpp

```diff
@@ -10,8 +10,13 @@
 #ifdef _IMAGE_TIFF
 #include "ImageTIFF.h"
 
+// This corrects a conflict where both libtiff and opencv3 try to redefine uint64 and int64
 extern "C" {
-#include <tiffio.h>
```

Comment:
The only changes other than these lines in this file is to rename all instances of `int64` to `int64_tiff` so there is no name collision with openCV.